README.md

```diff
@@ -5,26 +5,41 @@ nvml: Non-Volatile Memory Library
 [![Build status](https://ci.appveyor.com/api/projects/status/1f5jwqpqs89itr2k?svg=true)](https://ci.appveyor.com/project/krzycz/nvml-10qrw)
 [![NVML release version](https://img.shields.io/github/release/pmem/nvml.svg)](https://github.com/pmem/nvml/releases/latest)
 
-This is the top-level README.md the NVM Library.
+This is the top-level README.md of the NVM Library.
 For more information, see http://pmem.io.
 
 ### The Libraries ###
 
-Please see the file [LICENSE](https://github.com/pmem/nvml/blob/master/LICENSE) for information on how this library is licensed.
+Please see the file [LICENSE](https://github.com/pmem/nvml/blob/master/LICENSE)
+for information on how this library is licensed.
 
 This tree contains a collection of libraries for using Non-Volatile Memory
-(NVM).  There are currently six libraries:
+(NVM).  There are currently eight libraries:
 
 * **libpmem** -- basic pmem operations like flushing
 * **libpmemblk**, **libpmemlog**, **libpmemobj** -- pmem transactions
 * **libvmem**, **libvmmalloc** -- volatile use of pmem
+* **libpmempool** -- persistent memory pool management
+* **librpmem** -- remote access to persistent memory (EXPERIMENTAL)
 
-These libraries are described in more detail on the
+and one command-line utility:
+
+* **pmempool** -- standalone tool for off-line pool management
```

Comment:
Well, *rpmemd* is not intended to be run by the user, so I was not sure if we should mention it here.